docs/modules/ROOT/pages/reference/extensions/datasonnet.adoc

```diff
@@ -0,0 +1,45 @@
+// Do not edit directly!
+// This file was generated by camel-quarkus-maven-plugin:update-extension-doc-page
+[id="extensions-datasonnet"]
+= DataSonnet
+:linkattrs:
+:cq-artifact-id: camel-quarkus-datasonnet
+:cq-native-supported: true
+:cq-status: Stable
+:cq-status-deprecation: Stable
+:cq-description: To use DataSonnet scripts for message transformations.
+:cq-deprecated: false
+:cq-jvm-since: 2.10.0
```

Comment:
Well, 3.0.0-M1 is indeed without DataSonnet, but milestone requirements are a bit relaxed, aren't they?

Reply:
M1 is just a milestone, that looks ok.

Maybe the situation would be different if we ever face something re-introduced in let's say 3.2.0. Maybe updating the cq-jvm-since would make sense then. I don't know.

Reply:
> It might still be true, unless there were a version out without datasonnet.

The dependency [has been released since 2.10.0]
(https://mvnrepository.com/artifact/org.apache.camel.quarkus/camel-quarkus-datasonnet). 

> Well, 3.0.0-M1 is indeed without DataSonnet, but milestone requirements are a bit relaxed, aren't they?

That's what I think too